modules/@apostrophecms/modal/ui/apos/mixins/AposDocsManagerMixin.js

```diff
@@ -270,6 +270,30 @@ export default {
         this.allPiecesSelection.isSelected = this.checked.length === this.allPiecesSelection.total ||
           (this.checked.length && this.maxReached());
       }
+    },
+    getDocModuleName(doc) {
+      // Don't assume the piece has the type of the module,
+      // this could be a virtual piece type such as "submitted-draft"
+      // that manages docs of many types
+      if (doc) {
+        return doc.slug.startsWith('/') ? '@apostrophecms/page' : doc.type;
+      }
+      return this.moduleName;
+    },
+    getContentChangedTypes(doc, types) {
+      const submitDraftType = '@apostrophecms/submitted-draft';
+      if (doc) {
+        const moduleName = this.getDocModuleName(doc);
+        return [
+          moduleName,
+          submitDraftType,
+          ...moduleName !== doc.type ? [ doc.type ] : []
```

Comment:
This line is specifically for pages. I make sure we get `@apostrophecms/page` as well as the page type like `default-page`. Because in relationships `AposDocsManager` the `moduleName` is the page type and not the generic `@apostrophecms/page`.